raiden/settings.py

```diff
@@ -45,12 +45,11 @@
 DEFAULT_MATRIX_KNOWN_SERVERS = {
     Environment.PRODUCTION: (
         "https://raw.githubusercontent.com/raiden-network/raiden-service-bundle"
-        "/96b7df9ed93a1d73f9f4ede8db4773c0045e1c3b/known_servers"
-        "/known_servers-production-v3.0.0-test.json"
+        "/master/known_servers/known_servers-production-v3.0.0.json"
```

Comment:
Hm maybe this was why the whitelist reloading didn’t work….

Reply:
yeah, the other PR wasn't merged yet